props/saber_fett263_buttons.h

```diff
@@ -3026,18 +3032,15 @@ SaberFett263Buttons() : PropBase() {}
       default:
         break;
       case MENU_PRESET:
-        if (direction > 0) {
-          if (SaberBase::IsOn()) {
+        if (SaberBase::IsOn()) {
+          if (direction > 0) {
             next_preset_fast();
           } else {
-            SelectPreset(current_preset_.preset_num + direction);
-          }
-        } else {
-          if (SaberBase::IsOn()) {
             previous_preset_fast();
-          } else {
-            SelectPreset(current_preset_.preset_num + direction);
           }
+        } else {
+          int preset = (current_preset_.preset_num + direction) % GetNumberOfPresets();
```

Comment:
I recommend replacing this with:

int num_presets = GetNumberOfPresets();
int preset = (current_preset_.preset_num + num_presets * 10 + direction) % num_presets;

Otherwise it might end up doing weird stuff with negative numbers.

Reply:
this seems to work differently, I can't actually dial to the final 2 presets using this

Reply:
the previous version worked without issue for normal and incremented turns in testing in both directions

Reply:
Nvm, that seems to skip the 1st two presets if I start by moving backwards first, need to figure out why

Reply:
OK, fixed and working now